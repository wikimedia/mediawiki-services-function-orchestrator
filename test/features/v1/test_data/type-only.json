{
	"Z1K1": "Z10005",
	"Z10005K1": "1"
}
