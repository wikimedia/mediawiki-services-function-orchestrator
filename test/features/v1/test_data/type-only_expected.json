{
	"Z1K1": {
		"Z1K1": "Z4",
		"Z4K1": "Z10005",
		"Z4K2": [
			"Z3",
			{
				"Z1K1": "Z3",
				"Z3K1": "Z6",
				"Z3K2": "Z10005K1",
				"Z3K3": {
					"Z1K1": "Z12",
					"Z12K1": [
						"Z1"
					]
				}
			}
		],
		"Z4K3": {
			"Z1K1": "Z8",
			"Z8K1": [
				"Z17",
				{
					"Z1K1": "Z17",
					"Z17K1": "Z99",
					"Z17K2": "Z831K1",
					"Z17K3": {
						"Z1K1": "Z12",
						"Z12K1": [
							"Z11",
							{
								"Z1K1": "Z11",
								"Z11K1": "Z1002",
								"Z11K2": "input to validate"
							}
						]
					}
				},
				{
					"Z1K1": "Z17",
					"Z17K1": "Z99",
					"Z17K2": "Z831K2",
					"Z17K3": {
						"Z1K1": "Z12",
						"Z12K1": [
							"Z11",
							{
								"Z1K1": "Z11",
								"Z11K1": "Z1002",
								"Z11K2": "schema for validation"
							}
						]
					}
				}
			],
			"Z8K2": "Z4",
			"Z8K3": [
				"Z20"
			],
			"Z8K4": [
				"Z14",
				{
					"Z1K1": "Z14",
					"Z14K1": "Z831",
					"Z14K4": {
						"Z1K1": "Z6",
						"Z6K1": "Z931"
					}
				}
			],
			"Z8K5": "Z831"
		}
	},
	"Z10005K1": "1"
}
